src/sentry/static/sentry/app/views/organizationIncidents/details/header.jsx

```diff
@@ -56,7 +57,7 @@ export default class DetailsHeader extends React.Component {
                 {t('Incidents')}
               </IncidentsLink>
               <Chevron src="icon-chevron-right" size={space(2)} />
-              {params.incidentId}
+              <Count value={params.incidentId} />
```

Comment:
I think it's nicer to not format IDs as counts

Reply:
Ah yeah, I think we should format it but not with `Count` (which shortens it using `k`, `m`, etc), but with `toLocaleString` instead.